sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.net6.0.cs

```diff
@@ -1594,6 +1594,7 @@ public partial class BlobBaseClient
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.BlobProperties>> GetPropertiesAsync(Azure.Storage.Blobs.Models.BlobRequestConditions conditions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.Storage.Blobs.Models.GetBlobTagResult> GetTags(Azure.Storage.Blobs.Models.BlobRequestConditions conditions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.GetBlobTagResult>> GetTagsAsync(Azure.Storage.Blobs.Models.BlobRequestConditions conditions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        protected static Azure.Core.TokenCredential GetTokenCredential(Azure.Storage.Blobs.Specialized.BlobBaseClient client) { throw null; }
```

Comment:
@schaabs @seanmcc-msft 

This actually follows the pattern that currently exists between `BlobServiceClient` and `BlobBatch` to pass the Authentication Policy to authenticate the `BlobBatchClient`
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.net6.0.cs#L173

Let me know what you think about this exposure, since I know we've done our best not to do this to pass our credentials.